Engine/Component/ComponentMeshRenderer.cpp

```diff
@@ -112,6 +125,7 @@ void ComponentMeshRenderer::AddDiffuseUniforms(unsigned int shader_program) cons
 	glUniform1i(glGetUniformLocation(shader_program, "material.diffuse_map"), 0);
 	glUniform4fv(glGetUniformLocation(shader_program, "material.diffuse_color"), 1, (float*)material_to_render->diffuse_color);
 	glUniform1f(glGetUniformLocation(shader_program, "material.k_diffuse"), material_to_render->k_diffuse);
+	//glActiveTexture(0);
```

Comment:
If this commented code is not usefull we should delete it to keep the code clean